Source/Initialization/WarpXInitData.cpp

```diff
@@ -105,6 +105,169 @@ WarpX::PostProcessBaseGrids (BoxArray& ba0) const
+    }
+    else if (current_deposition_algo == CurrentDepositionAlgo::Vay){
+      amrex::Print()<<"Current Deposition:   | vay" <<"\n";
+    }
+    else if (current_deposition_algo == CurrentDepositionAlgo::Esirkepov){
+      amrex::Print()<<"Current Deposition:   | esirkepov" <<"\n";
+    }
+    // Print type of particle pusher
+    if (particle_pusher_algo == ParticlePusherAlgo::Vay){
+      amrex::Print()<<"Particle Pusher:      | vay" <<"\n";
+    }
+    else if (particle_pusher_algo == ParticlePusherAlgo::Vay){
+      amrex::Print()<<"Particle Pusher:      | HigueraCary" <<"\n";
+    }
+    else if (particle_pusher_algo == ParticlePusherAlgo::Boris){
+      amrex::Print()<<"Particle Pusher:      | boris" <<"\n";
+    }
+    // Print type of charge deposition
+    if (charge_deposition_algo == ChargeDepositionAlgo::Standard){
+      amrex::Print()<<"Charge Deposition:    | standard" <<"\n";
+    }
+    else{
+      amrex::Print()<<"Charge Deposition:    | standard" <<"\n";
+    }
+    // Print field gathering algorithm
+    if (field_gathering_algo == GatheringAlgo::MomentumConserving){
+      amrex::Print()<<"Field Gathering:      | momentum-conserving" <<"\n";
+    }
+    else{
+      amrex::Print()<<"Field Gathering:      | energy-conserving" <<"\n";
+    }
+    // Print particle's shape factors
+    amrex::Print()<<"Particle Shape Factor:| " << WarpX::nox <<"\n";
+    amrex::Print()<<"-------------------------------------------------------------------------------"<<"\n";
+    // Print solver's type: Yee, CKC, ECT
+    if (WarpX::maxwell_solver_id == MaxwellSolverAlgo::Yee){
+      amrex::Print()<<"Maxwell Solver:       | yee\n";
+      }
+    else if (WarpX::maxwell_solver_id == MaxwellSolverAlgo::CKC){
+      amrex::Print()<<"Maxwell Solver:        | ckc" <<"\n";
+    }
+    else if (WarpX::maxwell_solver_id == MaxwellSolverAlgo::ECT){
+      amrex::Print()<<"Maxwell Solver:        | ect" <<"\n";
+    }
+  #ifdef WARPX_USE_PSATD
+    // Print PSATD solver's configuration
+    if (WarpX::maxwell_solver_id == MaxwellSolverAlgo::PSATD){
+      amrex::Print()<<"Maxwell Solver:       | psatd\n";
+      }
+    if ((m_v_galilean[0]!=0) or (m_v_galilean[1]!=0) or (m_v_galilean[2]!=0)) {
+      amrex::Print()<<"                      | - Galilean \n"<<
+      "                      |  - v_galilean=("<<m_v_galilean[0]<<","<<m_v_galilean[1]<<","<<m_v_galilean[2]<<")\n";
+      }
+    if (WarpX::update_with_rho==1) {
+      amrex::Print()<<"                      | - update with rho\n";
+      }
+    if (WarpX::do_dive_cleaning==1) {
+      amrex::Print()<<"                      | - div(E) cleaning\n";
+      }
+    if (WarpX::do_divb_cleaning==1) {
+      amrex::Print()<<"                      | - div(B) cleaning\n";
+      }
+    if (do_multi_J == 1){
+        amrex::Print()<<"                      | - multi-J deposition\n";
+        amrex::Print()<<"                      |   - J-deposition number="<<WarpX::do_multi_J_n_depositions<<"\n";
```

Comment:
Maybe we could rewrite this as
```suggestion
        amrex::Print()<<"                      |   - number of J depositions = "<<WarpX::do_multi_J_n_depositions<<"\n";
```
or we use the actual name of the input parameter, as we do elsewhere.